eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -112,8 +109,26 @@ public async Task<bool> InstallPlaywrightAsync()
         {
             try
             {
-                Console.WriteLine($"Installing Playwright to Browsers: {Environment.GetEnvironmentVariable("PLAYWRIGHT_BROWSERS_PATH")} Driver: {Environment.GetEnvironmentVariable("PLAYWRIGHT_DRIVER_PATH")}");
```

Comment:
Note, the "correct" way to install browsers is using the `Microsoft.Playwright.Cli` tool, however that package only works with TFM 5 (https://github.com/microsoft/playwright-dotnet/issues/1668). Even once it's upgraded to TFM 6, our CI environments are TFM 7 (and realistically will always be ahead of the general release version).

Reply:
It should be ok, the tool should be able to run in a higher TFM, it's likely also possible that previous TFMs are already installed

Reply:
> It should be ok, the tool should be able to run in a higher TFM

Any tips on how? It would be ideal to use the tool considering there are issues with using `npm`/`npx` in the environment as well:

```
Running 'npm install playwright'
Exception installing playwright: System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'npm' with working directory '/private/tmp/helix/working/AA53097D/w/B12B0925/e'. No such file or directory
   at System.Diagnostics.Process.ForkAndExecProcess(ProcessStartInfo startInfo, String resolvedFilename, String[] argv, String[] envp, String cwd, Boolean setCredentials, UInt32 userId, UInt32 groupId, UInt32[] groups, Int32& stdinFd, Int32& stdoutFd, Int32& stderrFd, Boolean usesTerminal, Boolean throwOnNoExec)
   at System.Diagnostics.Process.StartCore(ProcessStartInfo startInfo)
   at RunTests.ProcessUtil.RunAsync(String filename, String arguments, String workingDirectory, String dumpDirectoryPath, Boolean throwOnError, IDictionary`2 environmentVariables, Action`1 outputDataReceived, Action`1 errorDataReceived, Action`1 onStart, CancellationToken cancellationToken) in /private/tmp/helix/working/AA53097D/w/B12B0925/e/RunTests/ProcessUtil.cs:line 167
   at RunTests.TestRunner.InstallPlaywrightAsync() in /private/tmp/helix/working/AA53097D/w/B12B0925/e/RunTests/TestRunner.cs:line 114
```


> it's likely also possible that previous TFMs are already installed

Doesn't appear to be the case from the helix logs:

```
Installing Microsoft.Playwright.CLI dotnet tool to C:\h\w\B5E109F8\w\A488088E\e
Running 'C:\h\w\B5E109F8\p\dotnet-cli/dotnet tool install Microsoft.Playwright.CLI --tool-path C:\h\w\B5E109F8\w\A488088E\e'
You can invoke the tool using the following command: playwright
Tool 'microsoft.playwright.cli' (version '1.2.0') was successfully installed.
'C:\h\w\B5E109F8\p\dotnet-cli/dotnet tool install Microsoft.Playwright.CLI --tool-path C:\h\w\B5E109F8\w\A488088E\e' completed with exit code '0'
Installing Playwright Browsers: C:\h\w\B5E109F8\w\A488088E\e\ms-playwright
Running 'playwright install'
It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '5.0.0' (x64) was not found.
  - The following frameworks were found:
      7.0.0-alpha.1.21551.1 at [C:\h\w\B5E109F8\p\dotnet-cli\shared\Microsoft.NETCore.App]
      7.0.0-alpha.1.21567.1 at [C:\h\w\B5E109F8\p\dotnet-cli\shared\Microsoft.NETCore.App]

You can resolve the problem by installing the specified framework and/or SDK.

The specified framework can be found at:
  - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=5.0.0&arch=x64&rid=win10-x64
'playwright install' completed with exit code '-2147450730'

Displaying directory contents for C:\h\w\B5E109F8\w\A488088E\e\ms-playwright:
Exception in DisplayContents: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\h\w\B5E109F8\w\A488088E\e\ms-playwright'.
   at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path, Boolean ignoreNotFound)
   at System.IO.Enumeration.FileSystemEnumerator`1.Init()
   at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String directory, FindTransform transform, EnumerationOptions options, Boolean isNormalized)
   at System.IO.Enumeration.FileSystemEnumerableFactory.UserFiles(String directory, String expression, EnumerationOptions options)
   at System.IO.Directory.InternalEnumeratePaths(String path, String searchPattern, SearchTarget searchTarget, EnumerationOptions options)
   at RunTests.TestRunner.DisplayContents(String path) in C:\h\w\B5E109F8\w\A488088E\e\RunTests\TestRunner.cs:line 91
```

Reply:
cc/ @dougbu in case you have any suggestions here.

Reply:
You could try using the `DOTNET_ROLL_FORWARD: Major` environment variable to force it onto running with the current runtime version. This worked for me in some personal projects ([example](https://github.com/martincostello/alexa-london-travel-site/blob/b033a2619cc5530f6302f57e6e96c200d84c600d/.github/workflows/build.yml#L21)).

Reply:
Thanks @martincostello that did it!